### DISCUSSION TITLE
Split health into 2 versions of eacother

### DISCUSSION THREAD

**Discussion by Cboyjet4** [unitystation/unitystation#6982] [Ideas]
(((FEATURE SUGGESTION))) a suggestion, not something that'll be 100% likely to be implemented.
this is probably something that should be reserved for 1.0, just so that you don't have to constantly update it to be updated alongside unity station
i've returned to the unitystation github to make a suggestion.
-------------------------
I've noticed in my recent playthroughs that the health system (the innards of humans, and all those systems) like all the organs and stuff, they have actual physical properties (an example being the lungs ACTUALLY putting oxygen and all the air reagents into blood, and the liver having visible "filtered" stuff inside of it), needless to say the current health system is comparable to medical systems like brainmed, ABOVE that in some stuff, the current health system is extremely complex for the current gameplay, and simple features of "death" and damage and stuff, don't really do it justice right now, at least for this gameplay style for US13 currently, it has way more potential, so i come up with a solution.

Split the health system into 2 versions, the simple one, for more arcadey servers, examples being usually,
TG based servers. 
This health system should be based on the 2017 health system, so it could be easily convertable into the modern TG one, it could also be transformed into more complex. but still simple medical systems, like VGmed or oldbaymed

And the complex one (kinda current), for more HRP servers, take bay based servers for example, cev eris, bay itself, aurora, is12 ETC. that way, a medical system for both types of servers could be ready to go for servers that want a place on unitystation.

I feel like stuff like this could be done for most major feature changes, something like cqc as an example, we want to create a base for servers to fully branch off of, not a separate game.

This is just my suggestion though, it's just something i feel that could be really useful for servers that want to "move" to us13, end of suggestion.

**Comment by corp-0** [comment]
it would be a huge effort on us to do and support 2 health systems

**Comment by corp-0** [comment]
I do see the appeal of the suggestion though, it makes me think of other health system like ACE3 where they made sure all health components are perfectly modularized so the players can enable and disable parts of the simulation as they please.

**Comment by Cboyjet4** [comment]
> I do see the appeal of the suggestion though, it makes me think of other health system like ACE3 where they made sure all health components are perfectly modularized so the players can enable and disable parts of the simulation as they please.

Perfect, yeah like that, if tg wants to move, they gotta change a huge ammount, pratically everything, this would make general server moving a lot easier but of course, it's a huge task.

**Comment by Bod9001** [comment]
like, the only real difference between making this a old TG system vs a more advanced system, 
is just what can happen to those body parts, I feel like the current system emulates old TG system  quite well apart from the cuts in such, but you could just add a bool to disable those things, it would be relatively simple to have some extra code on the bones that break if they received some damage, that can be enabled or disabled from a bool

**Comment by Cboyjet4** [comment]
> like, the only real difference between making this a old TG system vs a more advanced system,
> is just what can happen to those body parts, I feel like the current system emulates old TG system quite well apart from the cuts in such, but you could just add a bool to disable those things, it would be relatively simple to have some extra code on the bones that break if they received some damage, that can be enabled or disabled from a bool

Sounds good, the health system is still 10 times more complex than OG tg though.
